cmd/otelcol/main.go

```diff
@@ -37,6 +39,7 @@ import (
 const (
 	ballastEnvVarName         = "SPLUNK_BALLAST_SIZE_MIB"
 	configEnvVarName          = "SPLUNK_CONFIG"
```

Comment:
We should really create `SPLUNK_CONFIG_FILE` to avoid confusion. Even in beta we may consider deprecating this -- or we could remove it. Perhaps we can do this in a separate PR, but then a GH issue should be filed.